corio.py

```diff
@@ -297,6 +302,9 @@ def main(options):
             terminated_tp = monitor_processes(processes)
             if terminated_tp:
                 test_ids = get_test_ids_from_terminated_workload(parsed_input, terminated_tp)
+                if receivers:
```

Comment:
We can add check  'if  senders and receivers;

Reply:
Done